### DISCUSSION TITLE
为什么我就一个文件也是显示排队

### DISCUSSION THREAD

**Discussion by john-hull1** [hiroi-sora/Umi-OCR#544] [Q&A]
我就一个文件也是显示排队，并且点开始了也是卡在第0页好久不动，谢谢解答
![image](https://github.com/hiroi-sora/Umi-OCR/assets/104374780/7a9fccfc-db1c-49a6-b360-3ba0f8ad5fed)

**Comment by hiroi-sora** [comment]
经过排查，是 **整页强制OCR** 导致的Bug。最新提交已修复该Bug。

由于该Bug影响面大，我紧急发布了修改后的程序包。可以重新下载 v2.1.2 版本，测试 **整页强制OCR** 的功能是否已经正常。

https://github.com/hiroi-sora/Umi-OCR/releases/latest

**Reply by john-hull1** [reply]
感谢回复，我换用2.1.2的rapid的版本后问题依然存在，但是换了您截图的2.1.1的rapid版本后问题得以解决

**Reply by hiroi-sora** [reply]
你的文件可以上传给我看看吗

**Reply by john-hull1** [reply]
[[OCR]_1-s2.0-S0304405X2400014X-mmc1_20240605_1955.layered.pdf](https://github.com/user-attachments/files/15586572/OCR._1-s2.0-S0304405X2400014X-mmc1_20240605_1955.layered.pdf)
[1-s2.0-S0304405X2400014X-mmc1.pdf](https://github.com/user-attachments/files/15586575/1-s2.0-S0304405X2400014X-mmc1.pdf)
[Documentation.pdf](https://github.com/user-attachments/files/15586578/Documentation.pdf)

**Reply by hiroi-sora** [reply]
我测试了你的文件，v2.1.2版，我这边都能正常处理这些文件。

![image](https://github.com/hiroi-sora/Umi-OCR/assets/56373419/e8a406c8-7a6d-407a-811c-2837f4f05135)

不过，你这两份文件都是纯文本文件（不是图片扫描件），所以不需要OCR哦，它原本就可以划选文字。

**Reply by john-hull1** [reply]
> 我测试了你的文件，v2.1.2版，我这边都能正常处理这些文件。
> 
> ![image](https://private-user-images.githubusercontent.com/56373419/336830401-e8a406c8-7a6d-407a-811c-2837f4f05135.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1OTA1OTIsIm5iZiI6MTcxNzU5MDI5MiwicGF0aCI6Ii81NjM3MzQxOS8zMzY4MzA0MDEtZThhNDA2YzgtN2E2ZC00MDdhLTgxMWMtMjgzN2Y0ZjA1MTM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDEyMjQ1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU5NTg0YzkwMTQ1YzA2ZjU1YWVlNTY5ZjU4ODdmMGUxYWQ4MmJiNmNkOGJhYmQ2OTZiNDY0MGYxYTgxNmQ0MmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bwuPK9zQi8mDjiSKYdVqevLy3aK-a0a3x4ebWSEwaSw)
> 
> 不过，你这两份文件都是纯文本文件（不是图片扫描件），所以不需要OCR哦，它原本就可以划选文字。

感谢回复，
我需要ocr这些文件，因为我在划线选择翻译软件翻译时翻译结果是乱码，我查询了相关解决办法说需要重新ocr
另，将其中一选项设置为页面强制ocr不知道您是否尝试过

**Reply by hiroi-sora** [reply]
> 将其中一选项设置为页面强制ocr

非常感谢您提供的信息！

经过排查，这确实是程序Bug。最新提交已修复该Bug。

由于该Bug影响面大，我紧急发布了修改后的程序包。你可以重新下载 v2.1.2 版本，测试 **整页强制OCR** 的功能是否已经正常。

https://github.com/hiroi-sora/Umi-OCR/releases/latest

**Reply by john-hull1** [reply]
> > 将其中一选项设置为页面强制ocr
> 
> 非常感谢您提供的信息！
> 
> 经过排查，这确实是程序Bug。最新提交已修复该Bug。
> 
> 由于该Bug影响面大，我紧急发布了修改后的程序包。你可以重新下载 v2.1.2 版本，测试 **整页强制OCR** 的功能是否已经正常。
> 
> https://github.com/hiroi-sora/Umi-OCR/releases/latest

您好，我重新下载尝试了2.1.2的paddle版本的，整页强制ocr功能正常，并且未出现之前的问题
